src/api/files.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const argCommand = require('../cmd-helpers').argCommand
+
+module.exports = send => {
+  return {
+    cp: argCommand(send, 'files/cp'),
+    ls: argCommand(send, 'files/ls'),
+    mkdir: argCommand(send, 'files/mkdir'),
+    stat: argCommand(send, 'files/stat'),
+    rm: function (path, opts, cb) {
```

Comment:
Ok, so the difference is that when deleting recursively, the dag links are also deleted from the repo.

anyway, if you agree, I think we should put these lines at the start of the function:

```javascript
if (typeof opts === 'function && !cb) {
  cb = opts
  opts = {}
}
```

This way `opts` can be omitted